tsconfig.json

```diff
@@ -5,10 +5,10 @@
     "emitDecoratorMetadata": true,
     "experimentalDecorators": true,
     "lib": ["es6", "dom", "es2018.promise", "es2017"],
-    "module": "es6",
+    "module": "es2020",
     "moduleResolution": "node",
     "sourceMap": true,
-    "target": "es5",
+    "target": "es6",
```

Comment:
This is just something tangential that I noticed. As we don't really support older browsers, we [can use es6](https://caniuse.com/?search=es6) as the target. This means that the code that is generated can be more compact.

Reply:
@SleeplessByte This is safe to do, right? We only support modern browsers.

Reply:
You also don't use typescript to "output" the code. You use babel (via webpack). So changing this doesn't change the output. 

The idea is good!